.travis.yml

```diff
@@ -10,16 +10,26 @@ osx_image: xcode9.1
 mono: none
 dotnet: 2.2.300
 
+env:
+- TEST_SUITE="without-cultures"
+- TEST_SUITE="cultures"
+
 before_install:
   - cd neo.UnitTests
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then ulimit -n 2048; fi
+script: |
+  if [[ "$TEST_SUITE" == cultures ]]; then 
+      dotnet restore
+      dotnet test
+  else
+      dotnet restore
+      find * -name *.csproj | xargs -I % dotnet add % package coverlet.msbuild
+      dotnet test --filter FullyQualifiedName!=Neo.UnitTests.UT_Culture.All_Tests_Cultures /p:CollectCoverage=true /p:CoverletOutputFormat=opencover
```

Comment:
`-v n` is needed?

How about export also as json? 
` \"json,opencover\"`

Reply:
what json format reports are used for?

Reply:
*  -v n. We can see the tests passed and time it takes. Good idea.
* What json format report used for?